content/sensu-core/1.6/api/silenced.md

```diff
@@ -246,22 +217,19 @@ Access-Control-Allow-Methods: GET, POST, PUT, DELETE, OPTIONS
 Access-Control-Allow-Origin: *
 Connection: close
 Content-length: 0
-
-$ curl -s -X GET http://localhost:4567/silenced | jq .
-[]
 {{< /highlight >}}
 
 #### API specification {#silenced-clear-post-specification}
 
 /silenced/clear (POST) | 
 -----------------------|------
-description            | Clear a silence entry.
+description            | Clear a single silence entry.
```

Comment:
A single specific silence entry?

Reply:
That might be too many modifiers, especially since "single" is already redundant.

How about: "Clear a single silence entry specified by its ID or by the applicable check and subscription"?